opensrp-app/src/main/java/org/smartregister/service/ImageUploadSyncService.java

```diff
@@ -40,6 +42,8 @@ protected void onHandleIntent(Intent intent) {
                                 + AllConstants.PROFILE_IMAGES_UPLOAD_PATH, profileImages.get(i));
                 if (response.contains("success")) {
                     imageRepo.close(profileImages.get(i).getImageid());
+                } else {
+                    Timber.e(new StringBuilder("Image Upload: could NOT upload image ID: ").append(profileImages.get(i).getImageid()).append(" PATH: ").append(profileImages.get(i).getFilepath()).toString());
```

Comment:
You could use Timber string formatting e.g `Timber.e("Image %s",path)"`

Reply:
Fixed